src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -802,6 +802,18 @@ internal override SyntaxNode AsInterfaceMember(SyntaxNode m)
                              .WithSemicolonToken(SyntaxFactory.Token(SyntaxKind.SemicolonToken))
                              .WithBody(null);
 
+                case SyntaxKind.OperatorDeclaration:
+                    var operatorDeclaration = (OperatorDeclarationSyntax)member;
+                    var abstractVirtualModifiers = operatorDeclaration.Modifiers.Where(x =>
+                        x.Kind() == SyntaxKind.AbstractKeyword
+                        || x.Kind() == SyntaxKind.VirtualKeyword
+                        || x.Kind() == SyntaxKind.PublicKeyword);
+                    return operatorDeclaration
+                        .WithModifiers(SyntaxTokenList.Create(abstractVirtualModifiers.ToArray()))
```

Comment:
is this .ToArray necessary?  if so, can you do SyntaxFactory.TokenList(abstractVirtualModifiers)

Reply:
done